### PR TITLE
Day 2 node agnostic

### PR DESCRIPTION
**Pull request by jackVanish**
Contains the code changes to make the day2 work agnostic (Actel parts aren't quite there yet but will be handled separately for cleanliness).

### CODE REVIEW

**Comment by CaitlinBate** [comment]
looks like this merge includes ACTEL code for multiple nodes - is that ready to be merged? i thought some of the nodes didnt work yet

**Comment by jackVanish** [comment]
It does; I was working on it in the same branch since it was all in the same ticket. I have been in the process of ironing them out but the issues I was running into are data issues. Specifically, the datasets we were using contained, in Hugos' words, "stations where no hydrophones were released." You (@CaitlinBate) and I were working on this a couple of weeks ago. Basically the data going in needs to change so that it won't confuse Actel when we try to run it. But in the instances where I did get it running (MIGRAMAR) I seem to recall it working fine. 

Bottom line, on re-investigating this, I don't think any of the bugs are with the code, but with the data going in and whether or not it will be suitable for Actel. So I feel OK about merging the in-progress work so far, especially since we need to get everything squared off ASAP.

**Comment by jackVanish** [comment]
@CaitlinBate Based on the above do you think we're good to merge this?

**Comment by CaitlinBate** [comment]
plan: take the erroring Actel stuff out into new branch to be dealt with. ACT and MIGRAMAR versions can be pushed
